tinker/faculty_bios/forms.py

```diff
@@ -79,6 +79,7 @@ class FacultyBioForm(Form):
         first = StringField('Faculty first name')
         last = StringField('Faculty last name')
         author_faculty = StringField("Faculty member's username", description="Enter your Bethel username.")
+        courseleaf_user = RadioField('', default='Yes', choices=[('Yes', 'Yes'), ('No', 'No')])
```

Comment:
You also need to add the hidden field to the forms. (not for the jev username, but for the other one). Also, make sure to test that for both views (admin and normal person)

Reply:
Fixed